### ISSUE TITLE
Work around a YugabyteDB regression querying the INFORMATION_SCHEMA.ATTRIBUTES table

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#14621]
YugabyteDB has a regression when querying `INFORMATION_SCHEMA.ATTRIBUTES`: https://github.com/yugabyte/yugabyte-db/issues/16081

We can work around it by removing the `ORDER BY` clause and ordering results in Java.

**Comment by lukaseder** [comment]
The failing query is:

```sql
select distinct udt_schema, udt_name
from information_schema.attributes
where udt_schema = 'public'
order by udt_schema, udt_name;
```

Perhaps, we shouldn't query that view in the first place and then remove duplicate values with `DISTINCT`, but run a query on `USER_DEFINED_TYPES` instead?

**Comment by lukaseder** [comment]
This query works:

```sql
select udt_schema, udt_name
from information_schema.attributes
where udt_schema = 'public'
group by udt_schema, udt_name
order by udt_schema, udt_name;
```

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.18.0, 3.17.9 (https://github.com/jOOQ/jOOQ/issues/14622), and 3.16.15 (https://github.com/jOOQ/jOOQ/issues/14623)
